pages/common/git-write-tree.md

```diff
@@ -0,0 +1,11 @@
+# git write-tree
+
+> Low level utility to create a tree object from the current index.
```

Comment:
Please provide a documentation link, as per the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
```suggestion
> Low level utility to create a tree object from the current index.
> More information: <https://git-scm.com/docs/git-write-tree>
```